src/server/models/listItem/providers/helpers.ts

```diff
@@ -279,3 +272,94 @@ export function getLanguagesRows(languagesProvided: string, queryString: string)
   }  || { rows: [] };
   return languageRows;
 }
+
+export function validateCountry(countryName: string): void {
```

Comment:
I don't think this needs to be thrown. FWIW - void functions are really difficult to test. It means you have to test for the exception. OR test if the function that calls validateCountry throws. Also, typescript really does not handle exception/throwing functions very well at all. It basically cannot detect if something throws, and therefore anyone using this function doesn't actually know if they should catch or handle it. 

It's different for statically typed languages, where you label a function to throw, and the compiler can tell you if there's an exception to handle. 

`return countriesList.find((country) => {
     return country.value === countryName;
   });` should suffice.

Reply:
useful points, thx @jenbutongit .  Adjusted this to return the country name and added a conditional in the calling function to avoid calling the DB if there's no match.